internal/pkg/synchronization/synchronizationjob.go

```diff
@@ -0,0 +1,28 @@
+package synchronization
+
+import (
+	"github.com/PhamDuyKhang/littledetective/internal/film"
+	"github.com/PhamDuyKhang/littledetective/internal/pkg/flog"
+)
+
+func Sync(filmID chan string, elService *film.FilmService, mongoService *film.FilmMongoService) {
+	logger := flog.New()
+	logger.SetLocal("synchronization")
+	for {
+		select {
+		case idFilm, ok := <-filmID:
+			if !ok {
+				return
+			}
+			tem, err := mongoService.GetFilmByID(idFilm)
+			if err != nil {
+			} else {
```

Comment:
What magic is this?

Reply:
I want a goroutines, which will run and wait a signal. This signal is data come form channel, and if a error is happen, the goroutiines can't shutdown, because next data will be lose, so i don't return, 
Currently I don't have any idea to handle this solution, if you have some idea, please give your idea and i try to improve my solution